src/uikit/InfoBox/InfoBox/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+
+import Column from 'uikit/Column';
+
+const Info = styled(Column)`
+  align-items: center;
+  justify-content: center;
+  border: 1px solid ${({ theme }) => theme.greyScale5};
+  font-family: ${({ theme }) => theme.fonts.default};
+  padding: 20px;
+  flex-basis: 0;
+  flex: 1;
+
+  ${({ styles }) => (styles ? styles : null)};
+`;
+
+const Value = styled('div')`
+  font-weight: 600;
+  font-size: 18px;
```

Comment:
@ciaranschutte Could we have all the typography (font sizes, weights, etc) in the theme?

Reply:
@nyanofthemoon I think we're moving towards keeping a lot of styling in local styled components; I'm going to leave this here for now